SPECS/tpm2-pytss/tpm2-pytss.spec

```diff
@@ -0,0 +1,68 @@
+Name:       tpm2-pytss
+Version:    1.2.0
+Release:    1%{?dist}
+Summary:    Python bindings for tpm2-tss
+License:    BSD
+URL:        https://github.com/tpm2-software/tpm2-pytss
+Group:      System Environment/Security
+Vendor:     Microsoft Corporation
+Distribution: Mariner
+
+Source0: https://github.com/tpm2-software/tpm2-pytss/archive/refs/tags/%{version}.tar.gz#/%{name}-%{version}.tar.gz
+%define sha512 %{name}=9f317b2144e27ac0e6b876d1987cd5c64f4c1f70d1689cea3d926749d9778b2ad813335269cff70c8335f15b72b1233fffe062f4cb5791c5480c58119cf42db7
+
+BuildRequires: python3-devel
+BuildRequires: python3-setuptools
+BuildRequires: python3-pkgconfig
+BuildRequires: python3-pycparser
+BuildRequires: python3-pip
+BuildRequires: git
+BuildRequires: tpm2-tss-devel
+
+BuildRequires: python3-packaging
+BuildRequires: python3-asn1crypto 
+BuildRequires: python3-cryptography
+BuildRequires: python3-setuptools_scm
+
+%if 0%{?with_check}
+BuildRequires: python3-pytest
+BuildRequires: python3-cryptography
+BuildRequires: python3-cffi
+BuildRequires: python3-pip
```

Comment:
I know this comes direct from Photon, but this also doesn't make sense here since it's already included outside the with_check. Since this comes from Photon it's probably best to simply leave it in for our package, but also I would really love to see an upstream Photon patch to fix it (assuming we'll continue pulling updates from them for this package).